start-site/src/main/resources/application.yml

```diff
@@ -1028,6 +1028,16 @@ initializr:
           description: Declarative REST Client. OpenFeign creates a dynamic implementation of an interface decorated with JAX-RS or Spring MVC annotations.
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-openfeign
+        - name: Spring Cloud LoadBalancer
+          id: cloud-loadbalancer
+          compatibilityRange: "[2.2.0.M4,2.2.x.BUILD-SNAPSHOT)"
```

Comment:
Is there a reason why the entry should be disabled with snapshots? (Compared to the rest of Spring Cloud I mean).

Reply:
No, for 2.2.x snapshots it should be enabled - if that's the result of it, then it's a bug; should this look differently to include 2.2.x snapshots?

Reply:
That code says the entry should be enabled beteen `2.2.0.M4` (inclusive) to the latest snapshot of the 2.2 line (excluded). We don't use such range but rather the version one should use at the bare minimum.

Reply:
ah, ok. thanks